docs/application/get-started/iot.md

```diff
@@ -2,15 +2,28 @@
 
 The Get Started Tutorial teaches you how to create basic Tizen applications for IoT devices.
 
+## Hardware Setup for IoT Development
+
+To start developing Tizen IoT applications, follow the [Get Started guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi 3 or 4 board.
+
 ## .NET Application
 
 The following guide demonstrates how to create and run a basic Tizen .NET application, which displays some text on the screen with no user interaction:
 
--   [IoT .NET application](../dotnet/get-started/iot/first-app.md)
+- [Create Your First Tizen IoT .NET Application](../dotnet/get-started/iot/first-app.md)
+
+	An IoT application is targeted to run on an IoT device. The applications can take advantage of a wide variety of functionalities available on IoT devices.
 
-    An IoT application is targeted to run on an IoT device. The applications can take advantage of a wide variety of functionalities available on IoT devices.
-    
 First, ensure that you have installed the latest version of Visual Studio Tools for Tizen. [**Learn more >**](../vstools/install.md)
```

Comment:
@surbhig12 I think you'd remember [Reni-90 's modification for get-started pages](https://github.com/Samsung/tizen-docs/pull/1129).

It seems that fix is not applied to this ".NET Application" section because of PR merging timing.
Would you please apply the style update separtely?

Reply:
I will fix them.